src/main/java/com/powsybl/openloadflow/network/VoltageControl.java

```diff
@@ -143,11 +143,19 @@ public boolean isHidden() {
         List<VoltageControl<?>> voltageControls = findVoltageControlsSortedByPriority(controlledBus);
         if (voltageControls.isEmpty()) {
             return true; // means all disabled
-        }
-        // we should normally have max 3 voltage controls (one of each type) because already merged
-        if (voltageControls.size() > 1) {
```

Comment:
This code is totally equivalent to previous one. if only one voltage control in the list it is necessary "this" (because we start from it) and so on "voltageControls.get(0) != this" is false.
I would revert this change.

Reply:
´This’ (imagine generator voltage control) could have all its controllers disabled, but the controlled bus can have another voltage control (imagine with transformer) with enabled controllers. In that case, the list of voltage control is of size 1. ´This’ is hidden for me.

Reply:
yes, agree